prefect_saturn/core.py

```diff
@@ -335,7 +363,7 @@ def _get_environment(
         self,
         cluster_kwargs: Dict[str, Any],
         adapt_kwargs: Dict[str, Any],
-    ) -> KubernetesJobEnvironment:
```

Comment:
Why is the type removed?

Reply:
because you're not guaranteed to be able to import `KubernetesJobEnvironment`. I wanted to be sure that this would work even in a future `prefect` 0.15 or 1.0 where `KubernetesJobEnvironment` is REMOVED (not just deprecated)

Reply:
Ah, makes sense